isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/referenzarchitektur/inhalt.adoc

```diff
@@ -468,6 +468,63 @@ Für derartige Aufrufe gibt es keine Vorgaben: Sie können aus beliebigen Schich
 [id="image-CallFAmGuX",reftext="{figure-caption} {counter:figures}"]
 image::blaupausen:referenzarchitektur/CallFAmGuX.png[align="center"]
 
+[[zentrale-sicherheitsaspekte]]
+=== Zentrale Sicherheitsaspekte in der Referenzarchitektur
+((vorher: Konzept Security: prinzipien-der-sicherheitsarchitektur))
+
+Die IsyFact-Referenzarchitektur beinhaltet auch Vorgaben für die Sicherheitsarchitektur und für die Prinzipien, nach denen sie aufgebaut ist.
+Eine davon ist die Forderung, dass IsyFact-konforme IT-Systeme gegen unberechtigte Nutzung zu schützen sind:
+
+* Authentifizierungsinformationen, wie Benutzername und Passwort, werden nicht über IT-Systeme hinweg weitergegeben.
+Nur der Eigentümer der Authentifizierungsinformationen und der zentrale IAM-Service besitzen Kenntnis über diese Informationen.
+* Eine Zweifaktor-Authentifizierung ist durch das Client-Zertifikat und dem darin enthaltenen Behördenkennzeichen (BHKNZ) als zusätzliches Authentifizierungsmerkmal konzipiert, allerdings setzt dies im IAM-Service voraus, dass dieses Merkmal dort auch ausgewertet werden kann.
+* Bei Aufrufen von Außen (durch Web-Clients oder externen Anwendungen) erfolgt die Authentifizierung und grobgranulare Autorisierung an den Außengrenzen der Anwendungslandschaft (Portal, Service-Gateway bzw. Authentication-Gateway).
+* Kommt der Aufruf von innerhalb der Systemlandschaft (z.B. bei einem Batch- oder Task-Aufruf), benötigt der (technische) Aufrufer eigene Authentifizierungsinformationen und die Authentifizierung und grobgranulare Autorisierung geschieht aus der Nutzungsschicht (Batch / Task) des aufrufenden IT-Systems.
+* Wird zur Ausführung des Aufrufs ein Service einer anderen Geschäftsanwendung / Domäne benötigt, erfolgt dessen Aufruf im Namen und mit der Autorisierung des ursprünglichen Aufrufers / Benutzers.
+* Die feingranulare Autorisierung geschieht an den Systemgrenzen / externen Schnittstellen der aufgerufenen Geschäftsanwendung.
+
+.Integration von IAM in eine Anwendungslandschaft
```

Comment:
Statt Anwendungslandschaft müssen wir "Systemlandschaft" schreiben, da wir in der software-technischen Sicht unterwegs sind.

Reply:
Innerhalb des Abschnitts ersetzt